Makefile

```diff
@@ -82,6 +82,14 @@ endif
 endif
 #end of engine selection block
 
+# add openssl headers for MacOS
+ifeq ($(shell uname),Darwin)
+	ifeq ($(CRYPTO_ENGINE_PATH),openssl)
+		CFLAGS += -I/usr/local/opt/openssl/include
+		LDFLAGS += -L/usr/local/opt/openssl/lib
```

Comment:
Don`t like absolute paths. Does these paths always the same?

Reply:
I wish I knew :)

Reply:
They are the same if user install openssl via homebrew, otherwise they should put their own path using `export` before make